src/app/components/ManualEntry.svelte

```diff
@@ -7,17 +8,16 @@
   export let tableData: WordListSource;
 
   /**
-   * Re-calculate word count
+   * Close table
    */
-  export let getLanguageSources: Function;
+  export let closeTable: Function;
 
   /**
-   * Disables being able to edit the title
-   * We no longer need this conditional if we work with IndexedDB IDs
+   * Re-calculate word count
    */
-  export let isEditingSource: boolean = false;
+  export let getLanguageSources: Function;
 
-  $: rowDataFromManualEntry = tableData.wordlist;
+  $: wordlist = tableData.wordlist;
```

Comment:
It might be best to avoid aliasing `tableData.wordlist` -- in other words, always use `tableData.wordlist`. :/